packages/common/src/services/Jupiter.ts

```diff
@@ -122,7 +124,8 @@ export const getJupiterQuoteByMint = async ({
     amount,
     slippageBps,
     swapMode,
-    onlyDirectRoutes
+    onlyDirectRoutes,
+    maxAccounts: MAX_ACCOUNTS
```

Comment:
ditto here not sure if we want to apply this to the entire app. seems probably safe to me tho, @rickyrombo wdyt?

Reply:
We should probably parameterize this one now to match the others